code/modules/client/preferences/migrations/tgui_prefs_migration.dm

```diff
@@ -39,6 +39,7 @@
 	migrate_antagonist(ROLE_REV_HEAD, list(ROLE_PROVOCATEUR))
 	migrate_antagonist(ROLE_TRAITOR, list(ROLE_SYNDICATE_INFILTRATOR, ROLE_SLEEPER_AGENT))
 	migrate_antagonist(ROLE_WIZARD, list(ROLE_WIZARD_MIDROUND))
+	migrate_antagonist(ROLE_CHANGELING, list(ROLE_STOWAWAY_CHANGELING))
```

Comment:
You don't need to do this because this was only for preferences that existed before the new prefs menu, when traitor meant all forms of traitor, etc. Because there was no latejoin changeling, nobody who picked changeling intended to play it. This should be treated as immutable and won't update existing players anyway.